save-backend/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=192.168.0.250:3306
+spring.datasource.username=root
+spring.datasource.password=123
```

Comment:
These properties can be removed from here

Reply:
Removed